R/locations_and_mascots.R

```diff
@@ -0,0 +1,152 @@
+               
+               # The rest that could'nt be covered with a group of other expressions
+               
+               name = gsub("Air Force", "Air_Force", name),
+               name = gsub("Bowling Green", "Bowling_Green", name), 
+               name = gsub("Brigham Young", "Brigham_Young", name),
+               name = gsub("Cal Poly", "Cal_Poly", name),
+               name = gsub("California Davis", "California_Davis", name),
+               name = gsub("California Riverside", "California_Riverside", name),
+               name = gsub("Central Michigan", "Central_Michigan", name),
+               name = gsub("Coastal Carolina", "Coastal_Carolina", name),
+               name = gsub("CSU Bakersfield", "CSU_Bakersfield", name),
+               name = gsub("Florida Atlantic", "Florida_Atlantic", name),
+               name = gsub("Florida Gulf Coast", "Florida_Gulf_Coast", name),
+               name = gsub("Georgia Southern", "Georgia_Southern", name),
+               name = gsub("Grand Canyon", "Grand_Canyon", name),
+               name = gsub("La Salle", "La_Salle", name),
+               name = gsub("Little Rock", "Little_Rock", name),
+               name = gsub("Loyola Marymount", "Loyola_Marymount", name),
+               name = gsub("Miami (Ohio)", "Miami_(Ohio)", name), 
+               name = gsub("Middle Tennessee", "Middle_Tennessee", name),
+               name = gsub("Notre Dame", "Notre_Dame", name),
+               name = gsub("Old Dominion", "Old_Dominion", name),
+               name = gsub("Oral Roberts", "Oral_Roberts", name),
+               name = gsub("Purdue Fort Wayne", "Purdue_Fort_Wayne", name),
+               name = gsub("Santa Clara", "Santa_Clara", name),
+               name = gsub("Seton Hall", "Seton_Hall", name),
+               name = gsub("Stony Brook", "Stony_Brook", name),
+               name = gsub("Texas A&M", "Texas_A&M", name), 
+               name = gsub("UMass Lowell", "UMass_Lowell", name),
+               name = gsub("USC Upstate", "USC_Upstate", name),
+               name = gsub("Utah Valley", "Utah_Valley", name),
+               name = gsub("Virginia Commonwealth", "Virginia_Commonwealth", name),
+               name = gsub("Wake Forest", "Wake_Forest", name))
+  
+  new_gsub_test <<- data.frame(df)
+  
+  return(df)
+  
+}
+
+
+name_location_mascot_function <- function(df){
+  
+  ## the first assignment sends dataframe to the nasty gsub function, in order to keep this main function clean
+  
+  df <- nasty_gsub_function(df)
+  
+  ## the next assignment a new column, location_mascots, that splits the name of each team into two elements
+  ## first element is the location of team, which doesn't contain any spaces in order to split the name into only two elements
+  ## second element is the rest of the words (regardless of the number of spaces) from the name, and contains the mascot
+  
+  df <- mutate(df, locations_mascots = str_split(name, " ", 2))
```

Comment:
Can you please make this sequence of `mutate()`s into a single `mutate()`?

Reply:
Just committed and pushed changes from the feedback on Friday. I thought I had pushed these already on Friday afternoon.